proto/viam/app/data/v1/data.proto

```diff
@@ -188,49 +188,53 @@ message DeleteBinaryDataByIDsResponse {
-// AddAnnotationsToBinaryDataByFileIDsRequest requests adding all specified annotations to each of the files when file IDs are provided
-message AddAnnotationsToBinaryDataByFileIDsRequest {
+enum Type {
+  ANNOTATION_TYPE_UNSPECIFIED = 0;
+  TAG_ANNOTATION = 1;
+}
+
+// AddTagsToBinaryDataByFileIDsRequest requests adding all specified tags to each of the files when file IDs are provided
+message AddTagsToBinaryDataByFileIDsRequest {
   repeated string file_ids = 1;
-  repeated Annotation annotations = 2;
+  repeated Annotation tags = 2;
 }
 
-// AddAnnotationsToBinaryDataByFileIDsResponse returns IDs of added annotations which can be passed to RemoveAnnotationsFromBinaryDataByFileIDs
-message AddAnnotationsToBinaryDataByFileIDsResponse {
-  repeated string annotation_ids = 1;
+// AddTagsToBinaryDataByFileIDsResponse returns nothing
+message AddTagsToBinaryDataByFileIDsResponse {
 }
 
-// AddAnnotationsToBinaryDataByFilterRequest requests adding all specified annotations to each of the files when a filter is provided
-message AddAnnotationsToBinaryDataByFilterRequest {
+// AddTagsToBinaryDataByFilterRequest requests adding all specified tags to each of the files when a filter is provided
+message AddTagsToBinaryDataByFilterRequest {
   Filter filter = 1;
-  repeated Annotation annotations = 2;
+  repeated Annotation tags = 2;
 }
 
-// AddAnnotationsToBinaryDataByFilterResponse returns IDs of added annotations which can be passed to RemoveAnnotationsFromBinaryDataByFileIDs
-message AddAnnotationsToBinaryDataByFilterResponse {
-  repeated string annotation_ids = 1;
+// AddTagsToBinaryDataByFilterResponse returns nothing
+message AddTagsToBinaryDataByFilterResponse {
 }
 
-// RemoveAnnotationsFromBinaryDataByFileIDsRequest requests removing the given annotations from each file when file IDs are provided
-message RemoveAnnotationsFromBinaryDataByFileIDsRequest {
+// RemoveTagsFromBinaryDataByFileIDsRequest requests removing the given tags value from each file when file IDs are provided
+message RemoveTagsFromBinaryDataByFileIDsRequest {
   repeated string file_ids = 1;
-  repeated string annotation_ids = 2;
+  repeated string value = 2;
 }
 
-// RemoveAnnotationsFromBinaryDataByFileIDsResponse returns the number of binary files which had annotations removed
-message RemoveAnnotationsFromBinaryDataByFileIDsResponse {
+// RemoveTagsFromBinaryDataByFileIDsResponse returns the number of binary files which had tags removed
+message RemoveTagsFromBinaryDataByFileIDsResponse {
   uint64 deleted_count = 1;
 }
 
-// RemoveAnnotationsFromBinaryDataByFilterRequest requests removing the given annotations from each file when a filter is provided
-message RemoveAnnotationsFromBinaryDataByFilterRequest {
+// RemoveTagsFromBinaryDataByFilterRequest requests removing the given tags value from each file when a filter is provided
+message RemoveTagsFromBinaryDataByFilterRequest {
   Filter filter = 1;
-  repeated string annotation_ids = 2;
+  repeated string value = 2;
```

Comment:
Naming this value to be consistent with how we define `Annotation` and not overload the term `tag`

Reply:
Since not exposing anything to do with an Annotation to the user right now, and making this API explicit about tags, would also make this `repeated string tags` throughout (and thus not need to specify 'tags value' in the message documentation above)

Reply:
Think this one still needs to be updated to `tags`

Reply:
You're right, all done